[2024-12-02T06:12:18.583+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-12-02T06:12:18.619+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: BuildELT_dbt.dbt_test manual__2024-12-02T06:11:47.454556+00:00 [queued]>
[2024-12-02T06:12:18.625+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: BuildELT_dbt.dbt_test manual__2024-12-02T06:11:47.454556+00:00 [queued]>
[2024-12-02T06:12:18.626+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2024-12-02T06:12:18.638+0000] {taskinstance.py:2330} INFO - Executing <Task(BashOperator): dbt_test> on 2024-12-02 06:11:47.454556+00:00
[2024-12-02T06:12:18.658+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=327) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-12-02T06:12:18.662+0000] {standard_task_runner.py:63} INFO - Started process 329 to run task
[2024-12-02T06:12:18.664+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'BuildELT_dbt', 'dbt_test', 'manual__2024-12-02T06:11:47.454556+00:00', '--job-id', '7', '--raw', '--subdir', 'DAGS_FOLDER/build_elt_with_dbt.py', '--cfg-path', '/tmp/tmpswe679zw']
[2024-12-02T06:12:18.668+0000] {standard_task_runner.py:91} INFO - Job 7: Subtask dbt_test
[2024-12-02T06:12:18.713+0000] {task_command.py:426} INFO - Running <TaskInstance: BuildELT_dbt.dbt_test manual__2024-12-02T06:11:47.454556+00:00 [running]> on host 196076bc5f60
[2024-12-02T06:12:18.784+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='BuildELT_dbt' AIRFLOW_CTX_TASK_ID='dbt_test' AIRFLOW_CTX_EXECUTION_DATE='2024-12-02T06:11:47.454556+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-12-02T06:11:47.454556+00:00'
[2024-12-02T06:12:18.788+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-12-02T06:12:18.790+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2024-12-02T06:12:18.792+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', '/home/***/.local/bin/dbt test --profiles-dir /opt/***/final_dbt --project-dir /opt/***/final_dbt']
[2024-12-02T06:12:18.807+0000] {subprocess.py:86} INFO - Output:
[2024-12-02T06:12:21.169+0000] {subprocess.py:93} INFO - [0m06:12:21  Running with dbt=1.8.7
[2024-12-02T06:12:22.293+0000] {subprocess.py:93} INFO - [0m06:12:22  Registered adapter: snowflake=1.8.4
[2024-12-02T06:12:23.113+0000] {subprocess.py:93} INFO - [0m06:12:23  [[33mWARNING[0m]: Configuration paths exist in your dbt_project.yml file which do not apply to any resources.
[2024-12-02T06:12:23.123+0000] {subprocess.py:93} INFO - There are 1 unused configuration paths:
[2024-12-02T06:12:23.125+0000] {subprocess.py:93} INFO - - models.final_dbt.example
[2024-12-02T06:12:23.349+0000] {subprocess.py:93} INFO - [0m06:12:23  Found 4 models, 1 snapshot, 2 sources, 459 macros
[2024-12-02T06:12:23.362+0000] {subprocess.py:93} INFO - [0m06:12:23
[2024-12-02T06:12:23.364+0000] {subprocess.py:93} INFO - [0m06:12:23  Nothing to do. Try checking your model configs and model specification args
[2024-12-02T06:12:24.559+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-12-02T06:12:24.567+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-12-02T06:12:24.609+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=BuildELT_dbt, task_id=dbt_test, run_id=manual__2024-12-02T06:11:47.454556+00:00, execution_date=20241202T061147, start_date=20241202T061218, end_date=20241202T061224
[2024-12-02T06:12:24.662+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-12-02T06:12:24.691+0000] {taskinstance.py:3498} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-02T06:12:24.695+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
